web/modules/custom/collection/src/Form/JoinCollectionForm.php

```diff
@@ -50,26 +50,48 @@ public function buildForm(array $form, FormStateInterface $form_state, AccountPr
     // confirmation form, disguised as a form submit button. The confirmation
     // form should open in a modal dialog for JavaScript-enabled browsers.
     if (Og::isMember($collection, $user)) {
-      $form['leave'] = [
-        '#type' => 'link',
-        '#title' => $this->t('Leave this collection'),
-        '#url' => Url::fromRoute('collection.leave_confirm_form', [
-          'rdf_entity' => $collection->id(),
-        ]),
-        '#attributes' => [
-          'class' => [
-            'use-ajax',
-            'button',
-            'button--small',
-            'button--blue-light',
-            'mdl-button',
-            'mdl-js-button',
-            'mdl-button--raised',
-            'mdl-js-ripple-effect',
-            'mdl-button--accent',
+      $form['leave_wrapper'] = [
+        '#type' => 'dropbutton',
```

Comment:
Hi @kalinchernev ,
I replaced it by mdl dropdown. 
Do we need also arrow (up/down) in the button? If yes, than probably I would need make changes in mdl repository otherwise it would be hard to get state of the dropdown for an arrow (open/closed).

I also noticed that some dropdowns have orange links and some blue. Maybe it also should be unified (ex. all blue)?